test/transformer/regression/arrays/IntArraysSymbolicLengthTest2.java

```diff
@@ -0,0 +1,34 @@
+package transformer.regression.arrays;
+
+import util.ArgVRandom;
+
+public class IntArraysSymbolicLengthTest2 {
+	public static void sort(int[] a) {
+	    final int N = a.length;
+	    for (int i = 1; i < N; i++) { // N branches
+	    	int j = i - 1;
+	    	int x = a[i];
+  // First branch (j >= 0):  2 + 3 + ... + N = N(N+1)/2 - 1 branches

```

Comment:
Looks like these are here to help explain the test. If this is the case can you make them a bit more comprehensive?